tests/phpunit/tests/block-bindings/wpBlockBindingsRegistry.php

```diff
@@ -13,10 +13,8 @@
  */
 class Tests_Blocks_wpBlockBindingsRegistry extends WP_UnitTestCase {
 
-	const TEST_SOURCE_NAME       = 'test/source';
-	const TEST_SOURCE_PROPERTIES = array(
-		'label' => 'Test source',
-	);
+	public static $test_source_name       = 'test/source';
+	public static $test_source_properties = array();
```

Comment:
I had to rename the properties to lowercase because PHPCS was complaining. 

And since I want to pass a function to `get_value_callback`, I had to put move
the initialization to inside of the `set_up()` method. Apparently you can't assign a closure to
a `CONST` or a static class property in PHP.

Reply:
Right, that is probably the best way to go.